gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -281,8 +285,6 @@ gst_tensor_query_serversink_render (GstBaseSink * bsink, GstBuffer * buf)
         mem = gst_buffer_peek_memory (buf, i);
         if (!gst_memory_map (mem, &map, GST_MAP_READ)) {
           nns_loge ("Failed to map memory");
-          gst_memory_unref (mem);
-          gst_buffer_unref (buf);
```

Comment:
Is this just bug patch? or related to the PR?

Reply:
It's the bug fix. I will separate the commit.